adapters/adview/adview.go

```diff
@@ -0,0 +1,167 @@
+package adview
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adViewAdapter struct {
+	EndpointTemplate template.Template
+}
+
+// Builder builds a new instance of the Adf adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+
+	urlTemplate, err := template.New("endpointTemplate").Parse(config.Endpoint)
```

Comment:
Nitpick: Consider using a consistent name for the template. Please choose use either `endpointTemplate` or `urlTemplate` and use the name consistently.

Reply:
fixed.